_posts/2022-11-29-November-Update-Exploring-V3D.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "November Update: Exploring V3D"
+date: 2022-11-28
+author: Maíra Canal
+layout: post
+permalink: /november-update-exploring-v3d/
+categories: Genel
+tags: [igalia, graphics, kernel]
+---
+
+After I posted my blog post, it has been a busy couple of months. As I pointed on my last blog post after I finished GSoC, I joined the [Igalia Coding Experience](https://www.igalia.com/coding-experience/) mentorship project, and also, in October, I traveled to Minneapolis for XDC 2022. So, let's make a summary of the last couple of months.
+
+# XDC 2022
+
+---
+
+Just a small thank you note to X.Org Foundation for sponsoring my travel to Minneapolis. XDC 2022 was a great experience, and I learned quite a lot during the talks. Although I was a beginner, all developers were very nice to me, and it was great to talk to experienced developers (and meet my mentors in person). Also, my talk is available on [YouTube](https://www.youtube.com/watch?v=nbRbM-Ld-44&t=3s&pp=ugMICgJwdBABGAE%3D).
+
+# Working with the Raspberry Pi 4
+
+---
+
+Although GSoC was a great learning experience, as I mentioned in my last blog post, I was willing to learn more about the Linux graphics stack. Fortunately, when I started the Igalia CE, [Melissa Wen](https://melissawen.github.io/) pitched me a project to increase IGT test coverage on DRM/V3D kernel driver. I was pretty glad to hear about the project as it allowed me to learn more about how a GPU works.
```

Comment:
As I mentioned in my last blog post, GSoC was a great learning experience and I'm willing to keep learning about the Linux graphics stack.